tests/test.yml

```diff
@@ -3,5 +3,6 @@
   remote_user: root
   vars:
     php_enable_webserver: false
+    php_cli_date_timezone: Europe/London
```

Comment:
No, the php_cli_date_timezone must be different from the php_date_timezone for the test.